crates/db_views_actor/src/comment_reply_view.rs

```diff
@@ -98,6 +107,7 @@ fn queries<'a>() -> Queries<
         comment_aggregates::all_columns,
         community_person_ban::community_id.nullable().is_not_null(),
         community_moderator::community_id.nullable().is_not_null(),
+        local_user::admin.nullable().is_not_null(),
```

Comment:
same here. this should be doable as a subquery which increases readability and modularity of the query and potentially increases perf since subqueries don't go against the postgresql join limits (e.g. geqo_threshold)